### PR TITLE
small bug fixes to make some scripts work with the new release

### PR DESCRIPTION
**Pull request by taroni**
I added 4 small bug fixes in few useful scripts.
@mcepeda

### CODE REVIEW

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by bucky-badger** [comment]
Can one of the admins verify this patch?

**Comment by dntaylor** [comment]
add to whitelist

File: PlotTools/rake/meta.rake [hunk 1]
```diff
@@ -82,7 +82,7 @@ namespace :meta do
     else
       # In MC, we can get the effective lumi from xsec and #events.
       file sample + '.lumicalc.sum' => sample + '.meta.json' do |t|
-        #sh "get_mc_lumi.py --sqrts #{sqrts} #{sample} `cat #{t.prerequisites} | extract_json.py n_evts` > #{t.name}"
+        sh "get_mc_lumi.py --sqrts #{sqrts} #{sample} `cat #{t.prerequisites} | extract_json.py n_evts` > #{t.name}"
```
Comment (mcepeda) [inline PlotTools/rake/meta.rake:85]: will this work? I thought we had deprecated the metadata, that is why I had commented out that line

Comment (taroni) [inline PlotTools/rake/meta.rake:85]: It worked for me. If it is deprecated,  I will look for another way to compute the mc lumi

Comment (mcepeda) [inline PlotTools/rake/meta.rake:85]: If you want to use it lets just un-deprecate it :). That's not a problem. 
(Can we for instance do a datadefs that is only used by LFV? That would rectify the issue that we had last time around with the datadef mainteinance)

Comment (taroni) [inline PlotTools/rake/meta.rake:85]: In the latest commit, I added the datadefs for LFVH  only.

**Comment by mcepeda** [comment]
@dntaylor , can we merge this?

**Pull request merged**